lib/doorkeeper/device_authorization_grant/oauth/device_authorization_request.rb

```diff
@@ -13,6 +13,7 @@ class DeviceAuthorizationRequest < Doorkeeper::OAuth::BaseRequest
         attr_accessor :host_name
 
         validate :client, error: :invalid_client
+        validate :client_supports_grant_flow, error: :unauthorized_client
```

Comment:
I noticed that other Doorkeeper grants also validate the requested scopes. I suppose we should do that too, even if our own applications don't use them.

Reply:
See also #9